### PR TITLE
SVC-381 gelf support

### PR DESCRIPTION
**Pull request by ian-axelrod**
Add gelf + muselog support

Requires major version increment, as including muselog (and potentially pygelf) restricts python version to 3.6+.

SVC-381

### CODE REVIEW

**Comment by ysimonson** [comment]
Not sure if it makes sense for this to be merged into master. I'd prefer master on our fork to be synced up with master on agschwender/pilbox, and then have a separate branch for the muse-specific customizations.

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +14,7 @@ before_install:
 install:
   - pip install tornado==4.5.1 Pillow==2.9.0 coveralls
```
Comment (sbly) [inline .travis.yml]: Can we add these to requirements.txt?

File: pilbox/app.py [hunk 2]
```diff
@@ -209,7 +210,7 @@ def fetch_image(self):
             raise tornado.gen.Return(resp)
         except (socket.gaierror, tornado.httpclient.HTTPError) as e:
             logger.warn("Fetch error for %s: %s",
-                        self.get_argument("url"),
+                        url,
```
Comment (sbly) [inline pilbox/app.py:213]: why was this changed?

**Review by sbly — APPROVED** [review]

**Comment by ian-axelrod** [comment]
@ysimonson ok, I'm fine w/ changing the base.

**Comment by ian-axelrod** [comment]
Done

**Review by ysimonson — APPROVED** [review]

File: pilbox/app.py (hunk 2 above)
Comment (ian-axelrod) [reply pilbox/app.py:213]: Check commit message:
`Will make debugging fetch issues a lot easier`

It was having a hell of a time debugging an issue w/ implicit_base_url earlier yesterday because I never got any log information that contained the base url in it, so I could not be 100% positive whether the setting was being used. It mislead me into thinking it was not.

Comment (ysimonson) [reply pilbox/app.py:213]: Looks like the old code was incorrect

File: .travis.yml (hunk 1 above)
Comment (ian-axelrod) [reply .travis.yml]: ok

Comment (ian-axelrod) [reply .travis.yml]: done
